case-uco/java/test/org/sleuthkit/caseuco/TestSuite.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.sleuthkit.caseuco;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+/**
+ *
+ * Runs all regression tests and contains utility methods for the tests The
+ * default ant target sets properties for the various folders.
+ */
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+    FacetDeserializerTests.class
+})
+public class TestSuite {
+
```

Comment:
Is this class supposed to be empty?

Reply:
I was mimicking the test suite in TSK proper with this except I just needed it for an entry point so that it runs the test suites marked in the annotation.  I think the `DataModelTestSuite` has some static helper methods, but I didn't need them for the CASE UCO code.